api/src/db/DynamoBusinessesDataClient.test.ts

```diff
@@ -22,10 +22,12 @@ describe("DynamoBusinessesDataClient", () => {
 
   let client: DynamoDBDocumentClient;
   let dynamoBusinessesDataClient: BusinessesDataClient;
+  let logger: LogWriterType;
   const formationDate = dayjs().subtract(3, "year").add(1, "month").day(1).format("YYYY-MM-DD");
   const naicsCode = "12345";
   const industry = "test-industry";
   const encryptedTaxId = "test-id-12345";
+  const errorId = "some-id";
```

Comment:
It looks like a lot of the tests in here are testing both the positive and the negative test cases in a single test. Ex: `finds a business by the businessName` is first checking that a business doesn't exist in the db and then later on does find a business. I'd like to see us explicitly define the negative cases so that we're defining what the negative behavior should be in the test name.

Reply:
Fixed.